File: Data Structures & Algorithms/longest-consecutive-sequence/submission-1.py
class Solution:
    def longestConsecutive(self, nums: List[int]) -> int:
        num =set(nums)
        count = 0
        for i in nums:
            if (i-1) not in num:
                length = 0
                while(i+length) in num:
                    length+=1
                count=max(count,length)
        return count

File: Data Structures & Algorithms/majority-element/submission-1.java
class Solution {
    public int majorityElement(int[] nums) {
        HashMap<Integer, Integer> map = new HashMap<>();
        for(int num : nums){
            map.put(num, map.getOrDefault(num, 0)+1);
        }
        int j = 0;
        int i = 0;
        for(Map.Entry<Integer,Integer> m: map.entrySet() ){
            if(m.getValue() > j && m.getValue() > nums.length / 2){
                j = m.getKey();
                i = m.getValue();
            }
        }
        return j;
    }
}